### PR TITLE
Flash the screen on climax

### PR DESCRIPTION
**Pull request by qwertyquerty**
## About The Pull Request

Flashes the screen on climax for 2 seconds

## Why It's Good For The Game

Really guys? This wasn't a thing already? Come on...

## Changelog
:cl:
fix: Flash the screen on climax
/:cl:

### CODE REVIEW

**Review by ghost — APPROVED** [review]
I am BeloneX and I approve of this PR.

**Review by Archanial — APPROVED** [review]
I am Archanial and I approve this

**Comment by Kerbin-Fiber** [comment]
I am Kerbin-Fiber and I approve of this PR.

**Review by Kerbin-Fiber — APPROVED** [review]

**Comment by katebishop49** [comment]
~~im totally not being help at gunpoint~~  i aprove

**Comment by MrJWhit** [comment]
0/10 not a custom flash picture, I want a blaring wall of those anime people orgasming.

File: code/modules/arousal/arousal.dm [hunk 1]
```diff
@@ -68,6 +68,10 @@
 		R.reaction(turfing ? target : target.loc, TOUCH, 1, 0)
 	if(!turfing)
 		R.trans_to(target, R.total_volume * (spill ? G.fluid_transfer_factor : 1))
+
+	overlay_fullscreen("flash", /obj/screen/fullscreen/flash) // To be more true to the hentai
+	clear_fullscreen("flash", 20)
```
Comment (PowerfulBacon) [inline code/modules/arousal/arousal.dm:73]: can't you just use flash_act and set visual to 1?

Comment (PowerfulBacon) [reply code/modules/arousal/arousal.dm:73]: actually nevermind that takes into account eye protection, as long as it has the crisp fading out

Comment (qwertyquerty) [reply code/modules/arousal/arousal.dm:73]: It does fadeout

**Review by ike709 — APPROVED** [review]

**Review by St0rmC4st3r — APPROVED** [review]
I am St0rmC4st3r and I approve these changes.

**Review by park66665 — APPROVED** [review]
idk about content but code looks okay

**Comment by qwertyquerty** [comment]
Meme pr??? Really?? This couldn't be more serious

**Review by silicons — APPROVED** [review]

**Comment by Hatterhat** [comment]
this is a certified funny moment

**Comment by qwertyquerty** [comment]
Guys it was a joke you weren't supposed to actually merge it

**Comment by PowerfulBacon** [comment]
its realistic though

**Comment by Hemlock32** [comment]
This literally makes it so masochists get randomly flashed mid fights... A meme PR that nerfs a neutral trait. Wow.

**Comment by ghost** [comment]
> This literally makes it so masochists get randomly flashed mid fights... A meme PR that nerfs a neutral trait. Wow.

It's not like he had no idea and just made this PR as a joke.

**Comment by Hemlock32** [comment]
> > This literally makes it so masochists get randomly flashed mid fights... A meme PR that nerfs a neutral trait. Wow.
> 
> It's not like he had no idea and just made this PR as a joke.

Not even blaming the author. He said himself it wasn't supposed to be merged...

**Comment by ike709** [comment]
This kills tactical masochism.

**Comment by qwertyquerty** [comment]
> This literally makes it so masochists get randomly flashed mid fights... A meme PR that nerfs a neutral trait. Wow.

Dude just don't be a masochist, easy

**Comment by ike709** [comment]
Shouldn't a true masochist _enjoy_ being nerfed?

**Pull request merged**